### PR TITLE
target allocator don't run as root

### PR DESCRIPTION
**Pull request by jaronoff97**
**Description:**  Changes the TA image to match what is done for the main operator image

**Link to tracking Issue(s):**

- Resolves: #3378 

**Testing:** <Describe what testing was performed and which tests were added.>

**Documentation:** <Describe the documentation added.>

### CODE REVIEW

**Review by swiatekm — APPROVED** [review]

**Comment by toporek3112** [comment]
Thank you for the PR! All checks have passed. Can you merge this please?

**Comment by jaronoff97** [comment]
i need another approval from the ta-maintainers group (which we are going to get rid of #3344)

**Pull request merged**
